### DISCUSSION TITLE
Update dropwizard 2.0.24 to latest version 4.0.5

### DISCUSSION THREAD

**Discussion by Zaeem6100** [dropwizard/dropwizard#8230] [Development]
I am new to drop wizard Now the main problem there is not sufficient documentation on how to update Drop Wizard. what are the steps can any one explain to me.

**Comment by joschi** [comment]
Please check the upgrade notes for [Dropwizard 3.0.x](https://www.dropwizard.io/en/release-3.0.x/manual/upgrade-notes/upgrade-notes-3_0_x.html) and [Dropwizard 4.0.x](https://www.dropwizard.io/en/release-4.0.x/manual/upgrade-notes/upgrade-notes-4_0_x.html).

If you have more specific questions, please feel free to ask them here. 😄
